ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1042,29 +1057,35 @@ static PrjFS_Result HandleNewFileInRootNotification(
     const MessageHeader* request,
     const char* relativePath,
     const char* absolutePath,
+    const char* relativeFromPath,
     bool isDirectory,
     PrjFS_NotificationType notificationType)
 {
 #ifdef DEBUG
     cout
         << "HandleNewFileInRootNotification: "
-        << absolutePath
-        << " (root-relative: " << relativePath << ")"
+        << (absolutePath == nullptr ? "[NULL]" : absolutePath)
```

Comment:
Under what conditions would `absolutePath ` be `nullptr` in `HandleNewFileInRootNotification`?

Reply:
I see now how this can happen, see comments above where `HandleNewFileInRootNotification` is called.